api/reader.graphql

```diff
@@ -1,9 +1,25 @@
 "Data that can be used as proof to validate notices and execute vouchers on the base layer blockchain"
 type Proof {
-  "Validity proof for an output"
-  validity: OutputValidityProof!
-  "Data that allows the validity proof to be contextualized within submitted claims, given as a payload in Ethereum hex binary format, starting with '0x'"
-  context: String!
+  firstIndex: Int!
+
+  """Reads a single `Input` that is related to this `Proof`."""
+  inputByInputIndex: Input
+  inputIndex: Int!
+  lastInput: Int!
+
+  """
+  A globally unique identifier. Can be used in various places throughout the system to identify this single value.
+  """
+  nodeId: ID!
+
+  outputIndex: Int!
+  validityInputIndexWithinEpoch: Int!
```

Comment:
Is it flatten? Some reason for that?

Reply:
it is the last proposal from the node group

Reply:
and probably it's not a big deal because the contracts need this format.

Reply:
also, may it will change

Reply:
https://github.com/cartesi/rollups-node/pull/337/files#diff-7041aafed4cac335c7cfd177729a5063242905c2460c78d40a424e8875a7e450R259

Reply:
https://discord.com/channels/600597137524391947/1172230296066871407/1230258214625480816